### ISSUE TITLE
Resources: Links are not visible

### ISSUE THREAD

**Issue by RyanAJensen** [pulibrary/DSS#566]
### Steps to recreate
Visit /resources and run the Axe dev tools

### URL
http://localhost:3000/resources

### Issues
Links on each entry do not have visible link text

### Severity
High

### WCAG Success Criterion
-[ ] Links must have discernible text (10)

### WCAG Conformance level (A, AA, AAA)
A

### User impact
Users are unable to see link text.

### Recommended fix
To solve this problem, you need to fix at least (1) of the following:
-( ) Element does not have text that is visible to screen readers
-( ) aria-label attribute does not exist or is empty
-( ) aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
-( ) Element has no title attribute
And fix the following:
-( ) Element is in tab order and does not have accessible text

### Acceptance criteria
- [ ] I can successfully use this feature with Voice Over in Safari.
- [ ] I have checked this feature using one of the following browser extensions and confirmed that it does not add any level A or level AA issues:
    * axe dev tools
    * Lighthouse
    * Wave
- [ ] WCAG Success Criterion are resolved

### Notes
This applies to every entry under resources
